tests/core/pyspec/eth2spec/test/lightclient_patch/block_processing/test_process_sync_committee.py

```diff
@@ -114,6 +122,67 @@ def test_sync_committee_rewards(spec, state):
         assert state.balances[index] == pre_balances[index] + expected_reward
 
 
+@with_all_phases_except([PHASE0, PHASE1])
+@with_configs([MAINNET], reason="to create duplicate committee")
+@spec_state_test
+def test_sync_committee_rewards_duplicate_committee(spec, state):
+    committee = spec.get_sync_committee_indices(state, spec.get_current_epoch(state))
+    committee_size = len(committee)
+    active_validator_count = len(spec.get_active_validator_indices(state, spec.get_current_epoch(state)))
+
+    # Preconditions of this test case
+    # With mainnet config, where active validators are less than SYNC_COMMITTEE_SIZE,
+    # the committee members SHOULD be duplicate.
+    assert active_validator_count < spec.SYNC_COMMITTEE_SIZE
+    assert committee_size > len(set(committee))
+
+    yield 'pre', state
+
+    pre_balances = state.balances.copy()
+
+    block = build_empty_block_for_next_slot(spec, state)
+    block.body.sync_committee_bits = [True] * committee_size
+    block.body.sync_committee_signature = compute_aggregate_sync_committee_signature(
+        spec,
+        state,
+        block.slot - 1,
+        committee,
+    )
+
+    signed_block = state_transition_and_sign_block(spec, state, block)
+
+    yield 'blocks', [signed_block]
+    yield 'post', state
+
+    duplicate_count = {}
+    for i, x in enumerate(committee):
+        if i != committee.index(x):
+            if x not in duplicate_count:
+                duplicate_count[x] = 1
+            duplicate_count[x] += 1
```

Comment:
this seems correct but i'll note that we can also do something like:

```python
from collections import Counter

multiplicities = Counter(committee)
# later, when reward accounting
if index in committee:
    reward = ...
    expected_reward += reward * multiplicities[index]
```

Reply:
Good collection usage! Thanks 👍